src/svgSprite.ts

```diff
@@ -1088,6 +1088,20 @@ export const svgSprite = (): string => `
       <path d="M479.345 30.1343H486.158V38.3437C487.555 32.2303 492.184 29.6103 496.725 29.6103C497.991 29.6103 499.127 29.6976 500 29.916V36.1604C498.821 35.9421 497.904 35.9421 496.638 35.9421C489.52 35.9421 486.158 42.4485 486.158 50.4832V66.2034H479.345V30.1343Z" fill="white"/>
       <path d="M169.177 9.96006C172.976 9.96006 175.858 12.8858 175.858 16.5101C175.858 20.0909 172.932 23.0602 169.177 23.0602C165.465 23.0602 162.54 20.0909 162.54 16.5101C162.54 12.8421 165.465 9.96006 169.177 9.96006Z" fill="white"/>
     </symbol>
+
+    <symbol width="24" height="24" viewBox="0 0 24 24" fill="none" id="svg-activities-outlined">
```

Comment:
I use a similar checklist icon for the "Activities" tab on the Applet navigation, and I was able to use the existing `checklist-outlined` svg so you may not need to add this: https://github.com/ChildMindInstitute/mindlogger-admin/blob/448f87854392fc4e109906cbe47804b1a44604a7/src/svgSprite.ts/#L555